README.md

```diff
@@ -30,89 +30,89 @@ Want to watch a screencast 🎥 about it? Check out https://symfonycasts.com/fou
 
 ## How to contribute
 
-The test suite of this library needs one or more database, and static analysis needs to be ran on the smaller PHP version
-supported (currently PHP 7.2), then it comes with a full docker stack.
-
-### Install docker
-
-You must [install docker](https://docs.docker.com/engine/install/) and [install docker-compose](https://docs.docker.com/compose/install/)
-at first before running the tests.
-
-### Run tests
-
-The library is shipped with a `Makefile` to run tests.
-Each target will build and start the docker stack and install composer only if needed.
-
-```shell
-$ make help
-validate                       Run sca, full test suite and validate migrations
-test                           Run PHPUnit tests suite
-sca                            Run static analysis
-docs                           Generate documentation to docs/output
-database-generate-migration    Generate new migration based on mapping in Zenstruck\Foundry\Tests\Fixtures\Entity
-database-validate-mapping      Validate mapping in Zenstruck\Foundry\Tests\Fixtures\Entity
-database-drop-schema           Drop database schema
-composer                       Run composer command
-docker-start                   Build and run containers
-docker-stop                    Stop containers
-docker-purge                   Purge containers
-clear                          Start from a fresh install (use it for troubleshooting)
-```
+### Running the Test Suite
 
-Use double-dash to pass any PHPUnit options or arguments with `make`:
-```shell
-$ make test -- --stop-on-failure
-$ make test -- --filter FactoryTest
-# don't use "=" options value. ie: don't do this:
-$ make test -- --filter=FactoryTest
-```
+The test suite of this library needs one or more databases, then it comes with a docker compose configuration.
 
-Same syntax is available for composer:
-```shell
-$ make composer -- info symfony/*
-```
+> [!NOTE]
+> Docker and PHP installed locally (with `pgsql` & `mongodb` extensions) is required.
 
-#### Run tests in different environments
+You can start the containers and run the test suite:
 
-You can create a `.env` file to change the context in which tests will execute:
-```dotenv
-USE_ORM=1
-USE_ODM=1
-USE_DAMA_DOCTRINE_TEST_BUNDLE=1
-SYMFONY_REQUIRE=5.4.* # allowed values: 5.4.* | 6.0.* | 6.1.* | 6.2.*
-PHP_VERSION=8.0 # allowed values: 8.0 | 8.1 | 8.2
-PREFER_LOWEST=1 # force composer to request lowest dependencies
-```
+```bash
+# start the container
+$ docker compose up --build --detach
```

Comment:
I've never used the `--build` and `--detach` options. What is their purpose and is it required?

Reply:
`--detach` is the same as `-d` if you don't add it, docker runs in the terminal (and not detached mode) so you have to open a new terminal.

about the `--build` I think it's not needed anymore, and docker builds automatically (it was not the case long time ago, and I kept this habits :sweat_smile:)